pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[tool.poetry]
+name = "django-easy-audit"
+version = "1.3.6-a2"
+description = "Yet another Django audit log app, hopefully the simplest one."
+license = "GPL3"
+authors = ["Natán Calzolari <natancalzolari@gmail.com>"]
+readme = "README.md"
+homepage = "https://github.com/soynatan/django-easy-audit"
+repository = "https://github.com/soynatan/django-easy-audit"
+documentation = "https://github.com/soynatan/django-easy-audit/wiki"
+classifiers = [
+    "Environment :: Plugins",
+    "Framework :: Django :: 3.2",
+    "Framework :: Django :: 4.0",
+    "Framework :: Django :: 4.1",
+    "Framework :: Django :: 4.2",
+    "Framework :: Django :: 5.0",
+    "Framework :: Django",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+packages = [{include = "easyaudit"}]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+django = [
+    {version = ">=3.2,<5", python = "<3.10"},
+    {version = "~5", python = ">=3.10"},
```

Comment:
Does this restrict it to Django 3.2 to 5? Or would a v6 work with this? I am unfamiliar with the poetry syntax. Asking because upper version bounds are problematic: https://iscinumpy.dev/post/bound-version-constraints/

Reply:
I think you're correct, this isn't quite right. This would restrict Django to `>=5.0.0 <6.0.0`. Here's the docs:

https://python-poetry.org/docs/dependency-specification/#tilde-requirements

I was trying to match the Python version to the supported Django version table from here:

https://docs.djangoproject.com/en/5.0/faq/install/#what-python-version-can-i-use-with-django

With that in mind, I think this should likely be `{version = ">=3.2.9,<6", python = ">=3.10"}`. I'd have to test that, but I think it would work correctly. Thoughts?

Reply:
I propose to only do `>=3.2.9"` without an upper bound. This allows users to test/use this package with a later Django version if they want (without having to wait for an update of this package). See the article I linked above.